linkerd/app/core/src/admin/mod.rs

```diff
@@ -80,10 +83,12 @@ impl<M: FmtMetrics> Service for Admin<M> {
 impl<M: FmtMetrics + Clone + Send + 'static> svc::Service<Connection> for Accept<M> {
     type Response = ();
     type Error = hyper::error::Error;
-    type Future = Box<dyn Future<Item = Self::Response, Error = Self::Error> + Send + 'static>;
+    type Future = Pin<
+        Box<dyn std::future::Future<Output = Result<Self::Response, Self::Error>> + Send + 'static>,
+    >;
 
-    fn poll_ready(&mut self) -> Poll<(), Self::Error> {
-        Ok(().into())
+    fn poll_ready(&mut self, cx: &mut Context<'_>) -> task::Poll<Result<(), Self::Error>> {
```

Comment:
Yes, because some code in this module still uses `futures` 0.1's `Poll` type — it's used for the `hyper::Service` impls, because the hyper dependency hasn't been updated yet. We use `task::Poll` here to disambiguate them. I could add a comment or use renaming imports to make that clearer, but it should be a temporary state.